src/common/transformations/src/transformations/mlir/mlir_op.cpp

```diff
@@ -100,6 +101,9 @@ void prepareMLIRKernelWithoutWrapper(mlir::OwningOpRef<mlir::ModuleOp>& module,
             pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
             pm.addNestedPass<func::FuncOp>(createCSEPass());
 
+            // Rewrite shape ops in tensor/arith/etc
+            pm.addPass(createConvertShapeToStandardPass());
+
```

Comment:
will we need the same in the GC pipeline?

Reply:
Not if we only feed it with statically shaped batches. In any case, adding it should be a single-line change, so we can just add it.